### ISSUE TITLE
Different Angles Evoriso Attunement Perk does not drop logs

### ISSUE THREAD

**Issue by SlyCedix** [DarkPacks/SevTech-Ages#2678]
## Bug Report
When attuned for the Evorsio constellation in Astral Sorcery, the perk "Different Angles" allows you to use your pickaxe as a shovel, or axe, however logs do not drop when using this perk.

Observed with both a vanilla stone pickaxe, and a TiCon pickaxe.

## Expected Behaviour
Logs should be dropping when logs are broken with a pickaxe while you have the Different Angles perk.

## Possible Solution
I'm assuming this is related to the fact that logs are set not to drop unless the log is broken with an axe, perhaps you could expand this behavior to any tool, so that you still wouldn't get logs by breaking with your hand. This functionality could also be buried in the age 2 Astral Sorcery advancement line to ensure that people still need axes for Ages 0 and 1 and a majority of 2.

## Steps to Reproduce (for bugs)
1. Attune to Evorsio with Astral Sorcery
2. Get to rank 2 and unlock the Different angles perk
3. Break logs with any pickaxe

## Client Information
* Modpack Version: 3.0.7
* Java Version: 1.8.0_25
* Launcher Used: Twitch
* Memory Allocated: 4G
* Server/LAN/Single Player: Server
* Optifine Installed: No
* Shaders Enabled: No

## Server Information
* Java Version: 1.8.0_151
* Operating System: Windows 10
* Hoster/Hosting Solution: Personal server machine
* Sponge (Non-Vanilla Forge) Server: No
* Additional Content Installed: None

**Comment by justinrusso** [comment]
That is unfortunate... I've reported it to the dev. Also, you may want to update your Java version - its quite outdated.

**Comment by SlyCedix** [comment]
Yeah, I have java 10 installed, diddn't realize that MC was using a way older version until just now, just pruned all my old installs, so it should be fixed

**Comment by justinrusso** [comment]
You will want the latest version of Java 8. Java 9 or 10 is not supported by Forge.

**Comment by SlyCedix** [comment]
I see, thanks for the help
